### DISCUSSION TITLE
Google Pixel 8 no 5g

### DISCUSSION THREAD

**Discussion by sgruntx** [kyujin-cho/pixel-volte-patch#213] [General]
Hi, 
My experience with latest version of app is mixed. My carrier is Orange PL and VoLTE working without any issues, but there's no 5G or VoWiFi. I tried everything. Is there something I can do about it to work? On my previous Pixel 6 5G and VoWiFi works. Please help @kyujin-cho

**Comment by fillwithjoy1** [comment]
Okay, open `*#*#4636#*#*` and send a screenshot of the phone info page. This should help determine what's the issue
